### ISSUE TITLE
Loadbalancer ConnectionSettings are not changed with an update

### ISSUE THREAD

**Issue by AlexLast** [kubernetes/kops#1456]
We are trying to change the IdleTimeout for the bastions ELB, by adding the following via edit cluster:

```
  topology:
    bastion:
      bastionPublicName: bastion-kube-test.domain.com
      idleTimeoutSeconds: 3600
```

After this, we run an update as usual and are told of the changes that will be made and to use the --yes flag:

```
$ work/bin/kops update cluster --state s3://s3-bucket --name kube-test.domain.comI0112 09:51:45.179815    3604 populate_cluster_spec.go:235] Defaulting DNS zone to: ZGQJ6GJOGZN3
I0112 09:51:45.799959    3604 executor.go:91] Tasks: 0 done / 107 total; 35 can run
I0112 09:51:47.181574    3604 executor.go:91] Tasks: 35 done / 107 total; 23 can run
I0112 09:51:48.017393    3604 executor.go:91] Tasks: 58 done / 107 total; 32 can run
I0112 09:51:49.960605    3604 executor.go:91] Tasks: 90 done / 107 total; 10 can run
I0112 09:51:50.085309    3604 dnsname.go:102] AliasTarget for "bastion-kube-test.domain.com." is "bastion-kube-test-1856968568.us-west-2.elb.amazonaws.com."
I0112 09:51:50.213086    3604 dnsname.go:102] AliasTarget for "api.kube-test.domain.com." is "api-kube-test-630721677.us-west-2.elb.amazonaws.com."
I0112 09:51:50.587034    3604 executor.go:91] Tasks: 100 done / 107 total; 7 can run
I0112 09:51:50.943189    3604 executor.go:91] Tasks: 107 done / 107 total; 0 can run
Will modify resources:
  LoadBalancer/bastion.kube-test.domain.com
  	ConnectionSettings  	 {"IdleTimeout":300} -> {"IdleTimeout":3600}

Must specify --yes to apply changes
```

Running with the --yes flag gives us:

```
$ work/bin/kops update cluster --state s3://s3-bucket --name kube-test.domain.com --yes
I0112 09:52:01.226793    3609 populate_cluster_spec.go:235] Defaulting DNS zone to: ZGQJ6GJOGZN3
I0112 09:52:04.151261    3609 executor.go:91] Tasks: 0 done / 107 total; 35 can run
I0112 09:52:05.484367    3609 executor.go:91] Tasks: 35 done / 107 total; 23 can run
I0112 09:52:06.340474    3609 executor.go:91] Tasks: 58 done / 107 total; 32 can run
I0112 09:52:07.479290    3609 executor.go:91] Tasks: 90 done / 107 total; 10 can run
I0112 09:52:07.608515    3609 dnsname.go:102] AliasTarget for "bastion-kube-test.domain.com." is "bastion-kube-test-1856968568.us-west-2.elb.amazonaws.com."
I0112 09:52:07.760460    3609 dnsname.go:102] AliasTarget for "api.kube-test.domain.com." is "api-kube-test-630721677.us-west-2.elb.amazonaws.com."
I0112 09:52:08.115358    3609 executor.go:91] Tasks: 100 done / 107 total; 7 can run
I0112 09:52:08.464631    3609 executor.go:91] Tasks: 107 done / 107 total; 0 can run
I0112 09:52:08.464747    3609 dns.go:135] Pre-creating DNS records
I0112 09:52:09.218047    3609 update_cluster.go:188] Exporting kubecfg for cluster
Wrote config for kube-test.domain.com to "/home/ubuntu/.kube/config"
Kops has changed your kubectl context to kube-test.domain.com
```

However when checking in AWS the value hasn't been changed, I tried running a rolling-update incase it needed this for some reason which didn't work either. When running without the --yes flag it shows the same output as before:

```
Will modify resources:
  LoadBalancer/bastion.kube-test.domain.com
  	ConnectionSettings  	 {"IdleTimeout":300} -> {"IdleTimeout":3600}
```

Thanks!

**Comment by lewistuff** [comment]
Cheers for raising @AlexLast - I was just about to post the same issue. I'm seeing this with KOPS `1.5.0-alpha3`.

**Comment by justinsb** [comment]
This should be fixed by #1502 - silly mistake I made!

**Comment by AlexLast** [comment]
@justinsb Great, thanks!
